lisa/sut_orchestrator/azure/tools.py

```diff
@@ -38,6 +38,10 @@ def get_version(self) -> str:
         found_version = find_patterns_in_lines(result.stdout, [self.__version_pattern])
         return found_version[0][0] if found_version[0] else ""
 
+    def deprovision(self) -> None:
```

Comment:
add one optional parameter to decide deprovision + user or not?
-deprovision[+user]

Reply:
What means [+user]? Why an optional parameter is needed?

Is it the exact command? `waagent -deprovision[+user]`.

Reply:
waagent -deprovision
not remove current user
waagent -deprovision+user 
remove current user

Reply:
ok, will add it.

Reply:
tested it, after this command, the VM cannot be used again. In this transformer, keep the user. If it needs to export a clear vhd, we can add it later.